code/obj/machinery/door/door_control.dm

```diff
@@ -5,6 +5,8 @@
 	desc = "A remote control switch for a door."
 	var/id = null
 	var/timer = 0
+	var/cooldown = 0
```

Comment:
```suggestion
	var/cooldown = 0 SECONDS
```

Reply:
Speaking of which, would this be redundant to have it as seconds both on the var and the sleep function?

Reply:
yes. very.